common/cards/card-plugins/effects/_effect-card.js

```diff
@@ -3,28 +3,35 @@ import Card from '../_card'
 
 /**
  * @typedef {import('common/types/cards').EffectDefs} EffectDefs
+ * @typedef {import('common/types/cards').CardDefs} CardDefs
  * @typedef {import('common/types/cards').CardTypeT} CardTypeT
  * @typedef {import('models/attack-model').AttackResult} AttackResult
  * @typedef {import('utils').GameModel} GameModel
  */
 
 class EffectCard extends Card {
 	/**
-	 * @param {EffectDefs} defs
+	 * @param {EffectDefs & CardDefs} defs
```

Comment:
CardDefs should be included in EffectDefs, in the ts code. Like EffectDefs = CardDefs & {...